### DISCUSSION TITLE
Ideas for KodaDot discord improvements

### DISCUSSION THREAD

**Discussion by damskyftw** [kodadot/nft-gallery#3246] [KodAmbassadors & community stuff]
We would like to rework the overall structure of our discord server. This would consist of:

- removing obsolete roles
- adding new bot for role taking and levels
- cleanse the server of roles that has been taken long time ago and don’t serve a purpose
- revive it and make it somewhat interesting ( games, extra roles, leadering )

If you have any ideas how we could improve our discord, feel free to let us know!

**Comment by spacesmutje** [comment]
Let's get to the easier part first: adding stuff 💯 

 ## 🎲 Discord game bots 🎮 
///
[**Quiz Bot**](https://quizbot.xyz/), let's you run random quizzes out of a pool or, if you want a more peer group focused quiz, create your own ones. 

Of course there is a _Premium_ version as well, it's 2€ on patreon monthly [https://www.patreon.com/ihxyz](https://www.patreon.com/ihxyz).

**Summary about premium service**: QuizBot Premium includes many features that allow persistent users of the bot to create, play and edit quizzes in more detail - all for a small monthly price. While QuizBot's core features are kept free, these features are exclusive to premium users and more will be added in future.

💡 
Test your communities knowledge about KodaDOT, the recent articles, Twitter Spaces and events. Of course random quizzes also keep visitors entertained.

///
**[Fight Bot](https://top.gg/de/bot/777475402737385472)**, let's you have your own Battle Royale for glory! Last User Standing at it's finest.

ℹ️ 
Overall my favourite one. Only downer, you can't change it's profile pic, which is a pistol.

💡 
Definetly a fun event, the more the better. Team up, be a lonely wolf. Throw your frozen burrito or counter the attack with your lasersword. Winner can grab the weekly trophy, or an NFT or whatsoever.

///
to be continued...

**Reply by damskyftw** [reply]
Thanks! Those are nice ideas, we are experimenting with g.gartic now and we’ll see how that goes :)

**Comment by damskyftw** [comment]
Small update:

I’ve done some research in which i’ll continue, so far it looks like we’ll be implementing security mod, current possibility is:

- https://beemo.gg/

Support and feedback system:

- https://tickettool.xyz/home-display

Memes, currencies, fun stuff

- https://dankmemer.lol/

**Reply by damskyftw** [reply]
- [x] Beemo

- [x] Ticket tool

**Reply by spacesmutje** [reply]
Additionally:

https://giveawaybot.party/ to simply make random giveaways.

**Comment by damskyftw** [comment]
# Since we have couple updates on the way on our discord server, we’d like to inform you on what is going to be happening/change if it wouldn’t be ideal!

Currently, we have tons of roles, some of them are obsolete, some of them are old. New role ideas are these: 

## Continent roles(they are up now)
- Europe
Etc.

## Clickable Roles
- Developer role (will gain access to developer category)
- Collector
- Artist


## Earnable Roles (earn by opening a ticket with us)
- Code review guild (earn acces to code review guild channel)
- Writer guild (earn access to writer guild channel)
- QA guild (earn access to qa guild channel)
- KodaDot OG
- Moderator (people who can help with kicking/baning)

## Leveling Roles:
- Level 1-5 - Rookie
- Level 5-10 - Improving
- Level 10-15 - Almost there 
- Level 15-20 Advanced (unlocks private channels)
- Level 20+ - Master 


## New Category: Guilds

That will include:

- Writers
- QA guild
- Code reivew guild
- other incoming guilds


Community manager role will get erased, renamed to TEAM with current people.

For now, i’ve cleaned out as many as possible discord channels to keep it low. Check the current status, let me know if something else can be archived.

Moderator role will stand for current role (pulse rifle) so everyone can understand what they are and can do.

Next up would be to add some fun bots - to be continued

**Reply by spacesmutje** [reply]
Love it. And i would love to kick some Bots 😏
